web/templates/dropdown_widget.hbs

```diff
@@ -1,4 +1,4 @@
-<button id="{{widget_name}}_widget" class="dropdown-widget-button" type="button" {{#if disable_keyboard_focus}}tabindex="-1"{{/if}} name="{{widget_name}}">
+<button id="{{widget_name}}-widget" class="dropdown-widget-button" type="button" {{#if disable_keyboard_focus}}tabindex="-1"{{/if}} name="{{widget_name}}">
```

Comment:
@wizardishuman you need to refactor this throughout the code wherever this id is used.

Reply:
yupp I got it I 'll try  @sayyedarib

Reply:
I completed the task ! @sayyedarib  , @zulipbot